src/twisted/mail/maildir.py

```
@@ -237,7 +240,7 @@ def exists(self, user, memo=None):
             return lambda: self.startMessage(user)
         try:
             a = self.alias[user.dest.local]
-        except:
+        except Exception:
```

Could this be `except KeyError:`, or do we really intend to swallow all exceptions here?